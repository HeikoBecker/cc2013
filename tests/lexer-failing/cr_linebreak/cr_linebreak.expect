cr_linebreak_only.c:6:1: keyword int
cr_linebreak_only.c:6:5: identifier main
cr_linebreak_only.c:6:9: punctuator (
cr_linebreak_only.c:6:10: keyword void
cr_linebreak_only.c:6:14: punctuator )
cr_linebreak_only.c:6:16: punctuator {
cr_linebreak_only.c:7:2: keyword int
cr_linebreak_only.c:7:6: identifier x
cr_linebreak_only.c:7:7: punctuator ;
cr_linebreak_only.c:8:2: identifier x
cr_linebreak_only.c:8:4: punctuator =
cr_linebreak_only.c:8:6: identifier x
cr_linebreak_only.c:8:7: punctuator +
cr_linebreak_only.c:8:8: constant 1
cr_linebreak_only.c:8:9: punctuator ;
cr_linebreak_only.c:9:1: punctuator }
