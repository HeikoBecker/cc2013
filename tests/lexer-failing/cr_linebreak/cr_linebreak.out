cr_linebreak_only.c:1:132: keyword int
cr_linebreak_only.c:1:136: identifier main
cr_linebreak_only.c:1:140: punctuator (
cr_linebreak_only.c:1:141: keyword void
cr_linebreak_only.c:1:145: punctuator )
cr_linebreak_only.c:1:147: punctuator {
cr_linebreak_only.c:2:2: keyword int
cr_linebreak_only.c:2:6: identifier x
cr_linebreak_only.c:2:7: punctuator ;
cr_linebreak_only.c:3:2: identifier x
cr_linebreak_only.c:3:4: punctuator =
cr_linebreak_only.c:3:6: identifier x
cr_linebreak_only.c:3:7: punctuator +
cr_linebreak_only.c:3:8: constant 1
cr_linebreak_only.c:3:9: punctuator ;
cr_linebreak_only.c:4:1: punctuator }
